### DISCUSSION TITLE
Help with Apache Cloudstack On a Single Machine

### DISCUSSION THREAD

**Discussion by charlesweng** [apache/cloudstack#9111] [Q&A] [answered]
Subject: Assistance Required with Network Configuration for Adding KVM Host to CloudStack Management Server

Dear Apache CloudStack Community,
I am currently working on setting up a CloudStack environment and encountering issues with configuring the network interfaces for adding a KVM host. Here are the details of my setup:

* Base machine: CentOS 9
* KVM host: CentOS 7.9

Following the CloudStack documentation, I have configured the network interfaces inside the KVM host (CentOS 7.9) as specified. Specifically, I have added the cloudbr0 configuration and modified the eth0 configuration to bridge with cloudbr0. Despite this, I am unable to add the KVM host to the CloudStack management server. Additionally, the KVM host cannot reach the internet with the current network interface configuration.

Here is a summary of the steps I followed:
1. Created the cloudbr0 bridge configuration.
2. Modified the eth0 interface to bridge with cloudbr0.

Despite these configurations, the KVM host remains inaccessible from the CloudStack management server, and it does not have internet connectivity.

Could you please provide guidance on resolving these network configuration issues? And also, provide information about why the cloudstack management server could not add the KVM host.

Thank you for your assistance.

**Comment by Pearl1594** [comment, marked as answer]
@charlesweng You could use the following resource: https://github.com/shapeblue/hackerbook/blob/main/1-user.md to bring up an all-in-one setup.
